saltproc/saltproc.py

```diff
@@ -265,108 +500,166 @@ def write_mat_file(self):
         """
         ana_keff_boc = self.read_res(0)
         ana_keff_eoc = self.read_res(1)
+        not_in_lib = open('NOT_IN_LIB', 'w')
         matf = open(self.mat_file, 'w')
         matf.write('%% Step number # %i %f +- %f;%f +- %f \n' %
                    (self.current_step, ana_keff_boc[0], ana_keff_boc[1],
                     ana_keff_eoc[0], ana_keff_eoc[1]))
-        matf.write(self.mat_def + ' burn 1 rgb 253 231 37\n')
-        for iso in range(self.number_of_isotopes):
-            matf.write('%s\t\t%s\n' %
-                       (str(self.isoname[iso]), str(self.core[iso])))
+        for key, val in self.core.items():
+            if key == '':
+                continue
+            matf.write(self.mat_def_dict[key].replace('\n', '') + ' fix 09c 900\n')
+            for indx, isotope in enumerate(self.isozai):
+                # filter metastables
```

Comment:
make this its own function. Given a list of isos, return a list with no metastables.